src/ShareToken.js

```diff
@@ -22,6 +22,7 @@ const ShareToken = (props) => {
             variant: 'success',
           });
           localStorage.setItem('userToken', shareToken);
+          props.setToken(shareToken);
           history.push('/list');
```

Comment:
Love the fact that you're standardizing and bringing the token/update token code to one component. I noticed that in both the `Welcome` and `ShareToken` components, we seem to have `localStorage.setItem('userToken', shareToken/token)` in the same context. A chance for some refactoring perhaps?
Something like this in `App` would do the trick and we could pass it through the same props you're using to pass `setToken` to:
``` 
const updateToken = (newToken) => {
    setToken(newToken);
    localStorage.setItem('userToken', token);
  };
```

Reply:
Taking the advice given and asking questions on PRs - why am I on a team of such rockstars?

Reply:
Yes, @kkatoch89 that makes a lot of sense! I'll do that.